metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json

```diff
@@ -34,7 +34,7 @@
             "panelIndex": "14",
             "panelRefName": "panel_0",
             "title": "Open Tables, Files, Streams",
-            "version": "7.3.1"
+            "version": "8.0.0-SNAPSHOT"
```

Comment:
Are this versions significant? I mean, is it some kind of minimum required version? (ditto for the other changes below)

Reply:
Nope, this is the version that was used to create the dashboard / visualizations. Ideally Beats must use a released version and mot master, like I did in this PR by mistake. Maybe I should re-create it using a released version :thinking:

Reply:
I finally re-created the dashboard with 7.8.1